<?xml version="1.0"?>

<launch>

  <param name="robot_description" command="$(find xacro)/xacro.py '$(find mantis_model)/urdf/mantis_visual.urdf.xacro'" />
  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <param name="publish_frequency" type="double" value="50.0" />
  </node>

  <node pkg="mantis_model" type="joint_state_pub" name="joint_state_pub" />

</launch>
